e2e/app_test.go

```diff
@@ -132,6 +133,50 @@ func TestAppDashStat(t *testing.T) {
 	}
 }
 
+func TestAppProxy(t *testing.T) {
+
+	testcases := []struct {
+		name string
+		flag string
+	}{
+		{
+			name: "without no-verify-ssl flag",
+			flag: "",
+		},
+		{
+			name: "with no-verify-ssl flag",
+			flag: "--no-verify-ssl",
+		},
+	}
+	for _, tc := range testcases {
+		tc := tc
+		t.Run(tc.name, func(t *testing.T) {
+			var totalReqs int64 = 1
+
+			pxy := proxy{successReqs: 0, errorReqs: 0}
```

Comment:
After changing `proxy` to `httpProxy`, could you rename the variable name to `proxy` please?

Also there's no need to initialize `successReqs` and `errorReqs` here since zero value of these fields are already `0`.